.wokeignore

```diff
@@ -1 +1,2 @@
 scout/server/blueprints/login/static/aspirant.svg
+scout/demo/rnafusion_report_example.html
```

Comment:
This keeps triggering the woke check. I guess it would be best to fix it, but not here, since it's created by an external software.